server/build/root-module-relative-path.js

```diff
@@ -0,0 +1,22 @@
+// Next.js needs to use module.resolve to generate paths to modules it includes,
+// but those paths need to be relative to something so that they're portable
+// across directories and machines.
+//
+// This function returns paths relative to the top-level 'node_modules'
+// directory found in the path. If none is found, returns the complete path.
+
+const RELATIVE_START = 'node_modules/'
+
+// Pass in the module's `require` object since it's module-specific.
+export default (moduleRequire) => (path) => {
+  const absolutePath = moduleRequire.resolve(path)
```

Comment:
Added a comment about this. This is duplicating the previous behavior, which resolved `babel-runtime/package` and then stripped off the package.json part.